pkg/http/api.go

```diff
@@ -951,6 +958,66 @@ func (a *api) onTerminateWorkflow(reqCtx *fasthttp.RequestCtx) {
 	}
 }
 
+func (a *api) onRaiseEventWorkflow(reqCtx *fasthttp.RequestCtx) {
```

Comment:
Since this API is universal, please use the wrapped gRPC one to avoid re-implementing the business logic.

Example: https://github.com/dapr/dapr/pull/5970/files#diff-237f098772824bb7f3c25e36c2565bf147e4dfedb1077bb351595b0ba4b881bdR2778-R2807